README.md

```diff
@@ -7,7 +7,13 @@ Base docker image for Java Spring Boot applications running in containers, speci
 https://hub.docker.com/r/hmcts/cnp-java-base/tags/
 
 ## Features
-Uses the [Cloud Foundry Java Buildpack Memory Calculator](https://github.com/cloudfoundry/java-buildpack-memory-calculator) to dynamically tune the JVM based on application characteristics and container resource limits.  The idea is to avoid applications being 'OOM killed' by the container orchestrator.
+
+The latest version, openjdk-8u191-jre-alpine3.9-1.0 does NOT make use of the memory calculator, which has always been a 
```

Comment:
Pretty much this whole section doesn't belong in the README, it should go into release notes, commit message or an announcement